From e271e044aec485a80ba6218a4a4c4ec5c4de83da Mon Sep 17 00:00:00 2001
From: Mohit Sabharwal <mohit@cloudera.com>
Date: Fri, 5 Feb 2016 16:32:56 -0800
Subject: [PATCH 0488/1164] CDH-36978 : HIVE-12966 : Change some
 ZooKeeperHiveLockManager logs to debug (Mohit
 Sabharwal via Ashutosh Chauhan)

    Signed-off-by: Ashutosh Chauhan <hashutosh@apache.org>
    (cherry picked from commit ecf45b9ffdb424234b6ae8fb8ceea4ee30c3a0d4)

Change-Id: I7f299da362b4c1c289e31acc0af7152201991673
---
 .../zookeeper/ZooKeeperHiveLockManager.java        |   10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/ql/src/java/org/apache/hadoop/hive/ql/lockmgr/zookeeper/ZooKeeperHiveLockManager.java b/ql/src/java/org/apache/hadoop/hive/ql/lockmgr/zookeeper/ZooKeeperHiveLockManager.java
index 7c7a8d1..bb2c390 100644
--- a/ql/src/java/org/apache/hadoop/hive/ql/lockmgr/zookeeper/ZooKeeperHiveLockManager.java
+++ b/ql/src/java/org/apache/hadoop/hive/ql/lockmgr/zookeeper/ZooKeeperHiveLockManager.java
@@ -214,7 +214,10 @@ public void releaseLocks(List<HiveLock> hiveLocks) {
       for (int pos = len-1; pos >= 0; pos--) {
         HiveLock hiveLock = hiveLocks.get(pos);
         try {
-          LOG.info(" about to release lock for " + hiveLock.getHiveLockObject().getName());
+          if (LOG.isDebugEnabled()) {
+            LOG.debug("About to release lock for " +
+                hiveLock.getHiveLockObject().getName());
+          }
           unlock(hiveLock);
         } catch (LockException e) {
           // The lock may have been released. Ignore and continue
@@ -259,7 +262,10 @@ private String getLockName(String parent, HiveLockMode mode) {
 
   private ZooKeeperHiveLock lock (HiveLockObject key, HiveLockMode mode,
       boolean keepAlive, boolean parentCreated) throws LockException {
-    LOG.info("Acquiring lock for " + key.getName() + " with mode " + key.getData().getLockMode());
+    if (LOG.isDebugEnabled()) {
+      LOG.debug("Acquiring lock for " + key.getName() + " with mode " +
+          key.getData().getLockMode());
+    }
 
     int tryNum = 0;
     ZooKeeperHiveLock ret = null;
-- 
1.7.9.5

